examples/amp-fit-text.amp.html

```diff
@@ -0,0 +1,74 @@
+<!doctype html>
+<html ⚡>
+<head>
+    <script async custom-element="amp-fit-text" src="https://cdn.ampproject.org/v0/amp-fit-text-0.1.js"></script>
+    <meta charset="utf-8">
+    <title>amp-fit-text</title>
+    <link rel="canonical" href="https://amp.dev/documentation/examples/components/amp-fit-text/index.html">
+    <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+    <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+      <script async src="https://cdn.ampproject.org/v0.js"></script>
+      <style amp-custom>
+    :root {
+      --color-primary: #005AF0;
+      --color-text-light: #fff;
+      --space-1: .5rem;  /* 8px */
+    }
+    .square-small, .square-medium, .square-large {
+      display: inline-block;
+      margin: var(--space-1);
+      color: var(--color-text-light);
+      background-color: var(--color-primary);
+    }
+    .square-small {
+      height: 100px;
+      width:  100px;
+    }
+    .square-medium {
+      height: 200px;
+      width:  200px;
+    }
+    .square-large {
+      height: 300px;
+      width:  300px;
+    }
+      </style>
+  </head>
+  <body>
+    <!-- 
```

Comment:
Are these HTML comments necessary in the doc?

Reply:
I added them just for clarity and readability for anyone who would be looking at this file, but I can pull them out if they aren't necessary.